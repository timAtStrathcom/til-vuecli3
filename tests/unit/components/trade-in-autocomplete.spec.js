import '@/widget-setup'
import { shallowMount } from '@vue/test-utils'
import TradeInLeadsAutocomplete from '@/components/trade-in-autocomplete'

describe('trade-in-leads-autocomplete', () => {
  it('matches snapshot', () => {
    const tradeInLeads = shallowMount(TradeInLeadsAutocomplete)
    expect(tradeInLeads).toMatchSnapshot()
  })
})
