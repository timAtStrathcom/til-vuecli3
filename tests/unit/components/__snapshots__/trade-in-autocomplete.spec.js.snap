// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`trade-in-leads-autocomplete matches snapshot 1`] = `
<div class="field has-addons">
  <div class="control is-expanded">
    <input placeholder="Enter Year, Make, and Model" class="input">
  </div>
  <div class="control">
    <button class="button is-primary">
      <font-awesome-icon-stub icon="[object Object]"></font-awesome-icon-stub>
    </button>
  </div>
</div>
`;
